### PR TITLE
EMT job and assets added

### PR DESCRIPTION
**Pull request by Jamini**
EMT job added, backpack, uniform, and satchel added. Map spawns added.

See http://aurorastation.org/forums/viewtopic.php?f=18&t=1443&start=20 for details.

Medical Doctor slots reduced by 1
2 EMT slots added

EMT does not have surgery or virology, but gains maintenance and external airlocks. 
Fixed a typo with nursing interns, preventing them from showing as medical personal on the manifest.

### CODE REVIEW

**Comment by Serveris** [comment]
User committed their gravity simulation logs. (data/investigate/gravity.html)


Why?

**Comment by Jamini** [comment]
Unintended. Why is that even in the repo at all?

**Comment by Serveris** [comment]
Did you launch a test server? If so, then dream daemon automatically processes one as gravity set up is part of a new round being generated.

**Comment by Jamini** [comment]
I mean, the file wouldn't have been included if someone hadn't historically ticked it to add it to the repo. If anything, it probably should just be removed since it's just a log and not relevant to the codebase?

Of course I set up a test server. I just missed that file being included in the commit. If it needs to be recommitted (which is a pain. I hate cherry-picking) I can do so.

**Comment by skull132** [comment]
For some reason, even after flagging the .html in question as something to be avoided, it still runs. Perhaps my push didn't run properly. Regardless, you can delete the file, commit the change, and thw push should be fine. Even with the .html included, it should be fine.

File: code/game/jobs/job/medical.dm [hunk 1]
```diff
@@ -36,14 +36,44 @@
...
+/datum/job/emt
+	title = "Emergency Medical Technician"
+	flag = EMT
+	department_flag = MEDSCI
+	faction = "Station"
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "the chief medical officer"
+	selection_color = "#ffeef0"
+	access = list(access_medical, access_morgue, access_chemistry, access_genetics,access_maint_tunnels, access_external_airlocks)
```
Comment (skull132) [inline code/game/jobs/job/medical.dm]: Why access_chemistry? Just curious, it seems unnecessary and simply calls out for powergame.

**Comment by skull132** [comment]
Second note. This pull seems like it's on a collision with pull #391. If needed, I'll cancel this, pull into my own repo and bash out the diff.

File: code/game/jobs/job/medical.dm (hunk 1 above)
Comment (Jamini) [inline code/game/jobs/job/medical.dm]: That line isn't the access line we use here. There are two levels of access in code. minimal_access, and a more expanded access seen on some servers. I simply copied the access line from MD and removed surgery/virology.

+	minimal_access = list(access_medical, access_morgue, access_maint_tunnels, access_external_airlocks)

Is the access list that our EMTs would have with the current configuration settings.

**Comment by Jamini** [comment]
I don't see any potential collisions? The diff in the map shouldn't matter, since I used the mapmerge tool properly. The areas that are changed in both pulls do not overlap.

File: code/game/jobs/job/medical.dm (hunk 1 above)
Comment (Jamini) [inline code/game/jobs/job/medical.dm]: If anything, access_genetics should be removed from the non-minimal access and surgery should be re-added. Either way, the line isn't going to be used any time soon on Aurora.

Comment (skull132) [inline code/game/jobs/job/medical.dm]: Yup, I was reading it on my phone and got the lines mixed up. I was hoping that the comment didn't publish, but phones. Anyways, yes, it being there is fine.

**Comment by skull132** [comment]
Alright, that stuff aside, looks good. The .html isn't an issue, as said. So flagging it green for pulling.

**Pull request merged**